nni/runtime/tuner_command_channel/channel.py

```diff
@@ -6,11 +6,19 @@
 """
 
 from __future__ import annotations
+import json
+from typing import Union
 
 __all__ = ['TunerCommandChannel']
 
 from .command_type import CommandType
 from .websocket import WebSocket
+from .semantic_command import ReportMetricData, UpdateSearchSpace
+from .semantic_command import ImportData, TrialEnd
+from .semantic_command import NewTrialJob, SendTrialJobParameter
+from .semantic_command import NoMoreTrialJobs, KillTrialJob
+Command = Union[ReportMetricData, UpdateSearchSpace,
```

Comment:
This should be a base class instead of type hint Union.
I expect the base class to include common logic of converting to JSON and from JSON. And if a specific command has special converting logic different from others, it can override the method.